### PR TITLE
Fix proposal expiredInQ state

### LINKED ISSUE #128: Calling execute on a Queued proposal that has expired breaks the subgraph
It seemingly no longer updates it's index at at all after this

### LINKED ISSUE #135: IPFS error at travis..
mkdir: can't create directory '/data/ipfs-static': Permission denied
Service 'ipfs' failed to build: The command '/bin/sh -c mkdir -p /data/ipfs-static' returned a non-zero code: 1
The command "docker-compose up -d graph-node" exited with 1.

Master branch travis is failing now!
This come from latest PR merged #129 
Might related to travis update their machines...

### PR DESCRIPTION
**Pull request by orenyodfat**
add test

fix #128 

fix #135

### CODE REVIEW

File: test/integration/domain.spec.ts [hunk 1]
```diff
@@ -906,5 +906,29 @@ describe('Domain Layer', () => {
   tokensForStaker: null,
 });
 
+    const { proposalId: p2 } = await propose({
+  rep: 10,
```
Comment (tibetsprague) [inline test/integration/domain.spec.ts]: the indentation is weird here

**Comment by tibetsprague** [comment]
not sure what's going on with the IPFS stuff in the build but its pretty important to get this merged in so we can get Alchemy working again

**Comment by orenyodfat** [comment]
> not sure what's going on with the IPFS stuff in the build but its pretty important to get this merged in so we can get Alchemy working again

How this comment it related to this PR ? @tibetsprague

**Comment by tibetsprague** [comment]
The build is failing with what looks like ipfs related errors

**Comment by ben-kaufman** [comment]
@orenyodfat is it ready to be merged now?

**Comment by orenyodfat** [comment]
> @orenyodfat is it ready to be merged now?

not yet
I am doing some updates..

**Pull request merged**